.gitignore

```diff
@@ -1,7 +1,7 @@
 node_modules
 build
-results.xlsx*
-temp.xlsx
+*.xlsx
```

Comment:
I think that maybe we should actually commit old results files in the repository to be able to recreate the whole database again from scratch.

Reply:
Wrote a bit about it in the larger comment below!